vast.go

```diff
@@ -547,3 +546,8 @@ func MarshalVAST(vastXML *VAST) (string, error) {
 // Legacy support for misspelled interfaces
 var UnmarshallVAST = UnmarshalVAST
 var MarshallVAST = MarshalVAST
+
+// IsWrapper returns whether the Vast is in question is wrapper
+func IsWrapper(vast *VAST) bool {
```

Comment:
```suggestion
func (vast *VAST) IsWrapper() bool {
```
This might be a bit cleaner as a receiver function. Just so someone could call it like `vastXml.IsWrapper()` instead of `vast.IsWrapper(vastXml)`